src/test/get_pipeline_metadata_response_test.cpp

```diff
@@ -68,6 +68,21 @@ class GetPipelineMetadataResponse : public ::testing::Test {
     tensorflow::serving::GetModelMetadataResponse response;
     ConstructorEnabledModelManager manager;
 
+    std::function<bool(const tensorflow::TensorShapeProto&, const std::vector<int64_t>&&)> isShapeTheSame =
+        [](const tensorflow::TensorShapeProto& actual,
+            const std::vector<int64_t>&& expected) -> bool {
+        if (static_cast<unsigned int>(actual.dim_size()) != expected.size()) {
+            SPDLOG_ERROR("Unexpected dim_size. Got: {}, Expect: {}", actual.dim_size(), expected.size());
+            return false;
+        }
+        for (int i = 0; i < actual.dim_size(); i++) {
+            if (actual.dim(i).size() != expected[i]) {
+                SPDLOG_ERROR("Unexpected dim[{}]. Got: {}, Expect: {}", i, actual.dim(i).size(), expected[i]);
```

Comment:
Do we need logs in unit tests?

Reply:
Yes if those help to debug issues in tests (and they do :))